app/experimenter/nimbus-ui/src/components/PageSummary/Takeaways/TakeawaysEditor.tsx

```diff
@@ -0,0 +1,154 @@
+    setShowEditor: (state: boolean) => void;
+  };
+
+export const TakeawaysEditor = ({
+  isLoading,
+  conclusionRecommendations,
+  conclusionRecommendation,
+  takeawaysSummary,
+  setShowEditor,
+  onSubmit,
+  submitErrors,
+  setSubmitErrors,
+  isServerValid,
+}: TakeawaysEditorProps) => {
+  const defaultValues = { conclusionRecommendation, takeawaysSummary };
+  type DefaultValues = typeof defaultValues;
+
+  const {
+    isValid,
+    isSubmitted,
+    FormErrors,
+    formControlAttrs,
+    handleSubmit,
+    formMethods,
+  } = useCommonForm<TakeawaysEditorFieldName>(
+    defaultValues,
+    isServerValid,
+    submitErrors,
+    setSubmitErrors,
+    {},
+  );
+
+  const onClickCancel = useCallback(
+    () => setShowEditor(false),
+    [setShowEditor],
+  );
+
+  const handleSave = handleSubmit(async (data: DefaultValues) => {
+    if (isLoading) return;
+    await onSubmit(data);
+  });
+
+  const conclusionRadioAttrs = formControlAttrs(
+    "conclusionRecommendation",
+    {},
+    false,
+  );
+
+  return (
+    <section id="takeaways-editor" data-testid="TakeawaysEditor">
+      <h3 className="h4 mb-3 mt-4">
+        <Row>
+          <Col>Takeaways</Col>
+          <Col className="text-right">
+            <Button
+              onClick={handleSave}
+              disabled={isLoading}
+              variant="primary"
+              size="sm"
+              className="mx-1"
+            >
+              Save
+            </Button>
+            <Button
+              onClick={onClickCancel}
+              disabled={isLoading}
+              variant="secondary"
+              size="sm"
+              className="mx-1"
+            >
+              Cancel
+            </Button>
+          </Col>
+        </Row>
+      </h3>
+      <FormProvider {...formMethods}>
+        <Form
+          noValidate
+          onSubmit={handleSave}
+          validated={isSubmitted && isValid}
+          data-testid="FormTakeaways"
+        >
+          {submitErrors["*"] && (
+            <Alert data-testid="submit-error" variant="warning">
+              {submitErrors["*"]}
+            </Alert>
+          )}
+          <Form.Group as={Row}>
+            <Form.Group
+              as={Col}
+              className="flex-grow-0"
+              controlId="conclusionRecommendation"
+            >
+              <Form.Label
+                className={classNames("font-weight-bold", {
+                  "is-invalid": !!submitErrors["conclusion_recommendation"],
+                })}
+              >
+                Recommendation:
+              </Form.Label>
+              {conclusionRecommendations!.map((option, idx) => (
+                <Form.Check
+                  key={idx}
+                  type="radio"
+                  value={option!.value!}
+                  label={option!.label!}
+                  title={option!.label!}
+                  {...conclusionRadioAttrs}
+                />
+              ))}
```

Comment:
Something's malformed in here with the label, clicking any of the labels jumps it back up to the first option instead of selecting the relevant option.

Reply:
Looks like this needed an ID, should be fixed now after adding one